MvvmCross/Platforms/Uap/Views/MvxApplication.cs

```diff
@@ -118,9 +118,9 @@ protected virtual void OnNavigationFailed(object sender, NavigationFailedEventAr
             throw new Exception("Failed to load Page " + e.SourcePageType.FullName);
         }
 
-        private async void OnSuspending(object sender, SuspendingEventArgs e)
+        private async void OnEnteredBackground(object sender, EnteredBackgroundEventArgs e)
         {
-            var deferral = e.SuspendingOperation.GetDeferral();
+            var deferral = e.GetDeferral();
 
             var suspension = Mvx.IoCProvider.GetSingleton<IMvxSuspensionManager>();
             await Suspend(suspension);
```

Comment:
Before we accept this, we should tidy up the methods a bit. We're no longer calling "Suspend" from within the OnSuspending - this could be a breaking change for some developers.
I suggest we keep the OnSuspending handler so that we can grab the deferral and call Suspend method.
Have an additional OnEnteredBackground where we grab the deferral invoke an EnteredBackground method.
I also don't think we should be calling SaveAsync in the OnEnteredBackground given that we don't call RestoreAsync - this feels like we're breaking this pattern